src/frontend/src/app/myprofile/myprofile.component.ts

```diff
@@ -9,8 +10,12 @@ import { Router } from '@angular/router';
 })
 export class MyProfileComponent {
   player: PlayerData;
+  playerStats: any;
```

Comment:
These properties exist on PlayerData, so we should use the player variable instead of creating these state variables.

Reply:
Removed duplicated variables and used playerData 
![image](https://user-images.githubusercontent.com/25267171/215925401-95e74c8f-e317-45ec-8ca6-f710c76d6205.png)